database/definition-variablen.txt

```diff
@@ -0,0 +1,84 @@
+Variable
+
+# Approved
+* "yes": this shall be put online, it is approved
+* "no": this one is confirmed to happen but misses some important information
+* "nada": this one is not going to happen this year
+* "": keep on checking this if there are news
+
+# Datestart: YYYYMMDD
+# Datenend: YYYYMMDD
+
+# Label: the name we use in the list
+# Name: The name we use in the detailed view
+# Shortname: no use yet, usable in social media etc
+# Hashtag: to be used in social media
+# Homepage: URL
+# City: English City name (could be changed one day to an international id, let's see ...)
+# Country: the two-digit country code
+# Venue: Name of the venue
+# OSM-Link: OpenStreetMap link for the building or the coordinates
+
+# CoC-Link: 
+* URL: Link to the CoC
+* nada: no code of condut
+* "empty": keep on checking this if there are news
```

Comment:
do we need an extra field for that? The result is the same (CoC not available), and "nada" is an uncommon value

Reply:
I like to have the empy field for me to see that I did not yet check it and something to have the "false" value which no has exactly that expression: "false" instead of "nada" @weeman1337